.prettierrc.json

```diff
@@ -0,0 +1,17 @@
+{
+  "arrowParens": "avoid",
+  "bracketSpacing": true,
+  "htmlWhitespaceSensitivity": "css",
+  "insertPragma": false,
+  "jsxBracketSameLine": false,
+  "jsxSingleQuote": false,
+  "printWidth": 80,
+  "proseWrap": "preserve",
+  "quoteProps": "as-needed",
+  "requirePragma": false,
+  "semi": true,
+  "singleQuote": true,
+  "tabWidth": 2,
+  "trailingComma": "none",
+  "useTabs": false
+}
```

Comment:
I would try to use the default Prettier's config as much as possible here. I think we don't have any specific rules, e.g. commonly using single quotes in those files is rather by accident than by our strict internal rules or preferences.

Reply:
Default was this:
```json
{
  "arrowParens": "avoid",
  "bracketSpacing": true,
  "htmlWhitespaceSensitivity": "css",
  "insertPragma": false,
  "jsxBracketSameLine": false,
  "jsxSingleQuote": false,
  "printWidth": 80,
  "proseWrap": "preserve",
  "quoteProps": "as-needed",
  "requirePragma": false,
  "semi": true,
  "singleQuote": false,
  "tabWidth": 2,
  "trailingComma": "none",
  "useTabs": false
}
```

Reply:
The only thing that is different is the `singleQuote`. Do we want to adjust that?

Reply:
I'd go with the default double-quotes. We use it it Python as well and it works well because of the fewer number of escapes (see https://prettier.io/docs/en/rationale.html#strings). If others are the default, do we need to include them in the config file?

Reply:
When we you the defaults we don´t need a config file.